packages/rx_bloc_cli/mason_templates/bricks/lib_dev_menu/__brick__/lib/alice_instance.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:alice/alice.dart';
```

Comment:
Is there a reason to put that file in a root level? I would recommend moving it to the dev menu lib, since it's used only there from what I see